### PR TITLE
Add graphql api docs 

### PR DESCRIPTION
**Pull request by polandll**
Still waiting on last test for createKeyspaces. Not quite ready to publish yet, but I'm making the PR.

### CODE REVIEW

File: modules/developers-guide/pages/cql-using.adoc [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+// pip install cqlsh
+// ----
+
+== Using the CQL API using cqlsh
+
+Run `cqlsh` from the Docker image.
+
+[source,bash]
+----
+docker run --rm -it nuvo/docker-cqlsh cqlsh localhost --cqlversion=3.4.5
```
Comment (csplinter) [inline modules/developers-guide/pages/cql-using.adoc:59]: We should move this cmd down to this section and change it to:

Get the IP address of your Stargate docker container (note this example assumes that the Stargate container name is "stargate")
```
docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' stargate
```

Connect to your Stargate container using cqlsh
```
docker run --rm -it nuvo/docker-cqlsh cqlsh {ip-address} --cqlversion=3.4.5
```

File: modules/developers-guide/pages/graphql-using.adoc [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+----
+
+IMPORTANT: Notice that the key for this JSON token is different than the value that the
+generate token has. It is `x-cassandra-token`, not `auth-token`.
+
+You should see a return value of:
+
+[source, plaintext]
+----
+NEED THE VALUE AFTER createKeyspace is VALID
```
Comment (csplinter) [inline modules/developers-guide/pages/graphql-using.adoc:63]: need to update this

Comment (polandll) [reply modules/developers-guide/pages/graphql-using.adoc]: oops. I've fixed it.

Comment (polandll) [reply modules/developers-guide/pages/graphql-using.adoc]: That's another reason I need to put all code in partials/examples and just cross-reference.

File: modules/developers-guide/pages/cql-using.adoc [hunk 3]
```diff
@@ -33,37 +33,49 @@ include::quickstart:page$quick_start-rest.adoc[tag=getDockerImage]
 
 include::quickstart:page$quick_start-rest.adoc[tag=startDocker]
 
-=== Installing standalone cqlsh using Docker
+== Using the CQL API using cqlsh
+
+=== Using the Stargate docker image cqlsh
+
+Run `cqlsh` from the Docker image.
```
Comment (csplinter) [inline modules/developers-guide/pages/cql-using.adoc:40]: This should be

1. Pull the cqlsh docker image
```
docker pull nuvo/docker-cqlsh
```

2. Grab the IP of your Stargate container (note this assumes that the Stargate container has the name "stargate")
```
docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' stargate
```

3. Connect to your Stargate instance using cqlsh
```
docker run --rm -it nuvo/docker-cqlsh cqlsh {ip-address} --cqlversion=3.4.5
```

Comment (polandll) [reply modules/developers-guide/pages/cql-using.adoc]: understand now!

**Pull request merged**